modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/utils/SW360ReleaseAdapterUtils.java

```diff
@@ -46,6 +46,16 @@ public static void prepareRelease(SW360Release release, SW360Component component
         SW360ReleaseAdapterUtils.setCopyrights(release, artifact);
     }
 
+    public static boolean isValidRelease(SW360Release release) {
```

Comment:
correct me if I'm wrong, but doesn't it also need a componentId?

Reply:
If one patches a release, one just have to pass the `releaseId`. But for creation the `componentId` is necessary. Maybe that is more complicated then my implementation suggest ... :(

Reply:
Even the version and name are not necessary for an update...